### DISCUSSION TITLE
RunPCA

### DISCUSSION THREAD

**Discussion by MaribelSotoN** [satijalab/seurat#4578] [Q&A] [answered]
Hi seurat team,

I'm new in this R analysis, so I tried to run the seurat pipeline but I have an error to use Run PCA
The code and the error is in the atacced file.

Thank you!

I really appreciate your help I'm stuck
![RunPCA_error](https://user-images.githubusercontent.com/63123341/120869701-a3c21200-c55c-11eb-8636-0bae684daf2c.PNG)

**Comment by timoast** [comment, marked as answer]
The issue is that you have provided a Seurat object as the `features` parameter, instead of a vector of features to use. `FindVariableFeatures()` returns a Seurat object.

**Comment by MaribelSotoN** [comment]
Thank you!
